docs/src/xhtml/getstarted/index.xhtml

```diff
@@ -50,6 +51,50 @@
 						</script>
 					</figure>
 					-->
+				<h3>NPM version</h3>
+				<p>
+					Tradeshift UI is also available as an
+					<a href="https://www.npmjs.com/package/@tradeshift/tradeshift-ui" target="_blank">
+						npm package </a
+					>. You can simply install it in your project executing the next command
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						npm i @tradeshift/tradeshift-ui --save
+					</script>
+				</figure>
+				<p>
+					then don't forget to import <code>ts.js</code> and <code>ts.css</code> in your main
+					application entrypoint
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						require('@tradeshift/tradeshift-ui');
+						require('@tradeshift/tradeshift-ui/ts.css');
+					</script>
+				</figure>
+				<p>
+					there is a style helpers in the package, so you can import it somewhere in your
+					<code>shared styles module</code>
+				</p>
+				<figure data-ts="DoxMarkup">
+					<script type="text/html">
+						@import '~@tradeshift/tradeshift-ui/ts-runtime';
+					</script>
+				</figure>
```

Comment:
Maybe it does worth mentioning it's a `less` file, what do you think?

Reply:
fair enough! I will make a note